Client/Experiments/Experiments.swift

```diff
@@ -64,7 +64,30 @@ enum Experiments {
             }
         }
     }
-
+    
+    private static var studiesSetting: Bool? = nil;
+    private static var telemetrySetting: Bool? = nil;
```

Comment:
Could we make these instance variables/methods instead of statics?

Reply:
Hmm ya we definitely could! but that would have to ship with app services with the `Nimbus` class to be accessed through the `Experiments.shared` object - It's a very trivial change, but it would have to wait for an a-s release (and wait for the SPM/places-uniffication work to be landed first)

I think it's worth it to do the API right and have it ship through A-S first 🤷 Thoughts?

Reply:
Maybe even worth it to have it be cross-platform and introduce it in the udl for nimbus (I think this is what you meant in slack 😬) this way we make sure that as long as all the client end up triggering those APIs the opt-out mechanism is working properly

Reply:
Alrighty, we'll get the settings code to be statics for now, since adjusting the instance variable means changing the SDK itself (which works, just fine, but we want to land this quickly)

then we will tackle https://mozilla-hub.atlassian.net/browse/EXP-2200 later